apix-integrationtests/alfresco/src/main/java/eu/xenit/apix/tests/versionhistory/VersionHistoryServiceTestJavaApi.java

```diff
@@ -1,41 +1,44 @@
 package eu.xenit.apix.tests.versionhistory;
```

Comment:
I thought so too at first, so i setup my project in a clean environment and ran this test first.
I get the same error every time.
[Remark] System is not in RO-mode. It has 2 days left, or i load up the license and get the same issue.
I don't see why this error occurs since the apix just  does a Node conversion and passes it to the Alfresco api.
The NodeRef looks correct and is identical.
```
06180062 Access Denied.  The system is currently in read-only mode.
org.alfresco.service.transaction.ReadOnlyServerException: 06180062 Access Denied.  The system is currently in read-only mode.

_apix create version_
    @Override
    public void createVersion(NodeRef nodeRef, Map<String, Serializable> versionProperties) {
        alfrescoVersionHistoryService.createVersion(c.alfresco(nodeRef), versionProperties);
    }
```

Reply:
Solved this issue by changing versionHistoryService.createVersion(...) to alfrizcoVersionHistoryService.createVersion(...)